Code/GraphMol/DistGeomHelpers/Embedder.cpp

```diff
@@ -167,7 +167,7 @@ bool _minimizeWithExpTorsions(
     // check if the energy is low enough
     double planarityTolerance = 0.5;
     if (field2->calcEnergy() > improperAtoms.size()*planarityTolerance) {
-      //std::cerr << "planarity check failed: " << field2->calcEnergy()  << std::endl;
+      BOOST_LOG(rdWarningLog) << "Planarity check failed with energy = " << field2->calcEnergy() << ". Discard conformer." << std::endl;
```

Comment:
Though I agree that BOOST_LOG is the way to do it if there were a warning, I don't think there should be one.
This is an implementation detail of sorts, like the chirality check that we do, and doesn't need to be "shared" with the user.